### ISSUE TITLE
Ship in Neon Core

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#474]
There is a ship in the middle of neon core!! i tried to load old saves even tried to start new game still there :(
Tried to verify files in gamepass it does nothing and tells me no files in que ???

please help
Frank

ErrorCodeNotListed=none
VersionNumber=1.7.36.0

---

## Details 
I have been having this problem for over a month now and Bethasda support is useless

## Game Version
1.7.36.0

## Language
English (EN)

## New Game
Yes

## Reproduction Steps
Uninstalled game, formatted pc, new game new and old saves I go to neon ship is still there some kind of graphic file issue

**Comment by Pickysaurus** [comment]
I'm not sure how it got there, but this sounds like a bug in your save which I'm not sure can be fixed easily. If you can board the ship, perhaps you can add it to your fleet to remove it?

**Comment by Constellation-VASCO** [comment]
I can't actually go on it just through it :(  (blindly walk through it)

**Comment by Constellation-VASCO** [comment]
Yes I also think it has some thing to do with my saves. Unfortunately I don't have any saves before the issue :(

**Comment by Constellation-VASCO** [comment]
I started a completly new game and ship is gone. so it must be in the save files
